saturnbase-rstudio/Dockerfile

```diff
@@ -0,0 +1,81 @@
+FROM rocker/r-ver:4.1.0
```

Comment:
This is GPLv2 - not a dealbreaker, but means we'd need to license this image as GPLv2 as well.